src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -860,6 +866,18 @@ public void clickAddQuestionButton() {
         waitForPageToLoad();
     }
 
+    public void clickTemplateModalButton() {
+        scrollElementToCenterAndClick(addTemplateModalLoadButton);
```

Comment:
somehow I tried `click(addTemplateModalLoadButton)` multiple times previously but it doesnt work. It executes the function but no modal appears. I suspect that it is clicking on some space near the id so I used this function instead.

Reply:
This is what is done to select questions like `MCQ`, `MSQ` from dropdown.

```java
    public void selectNewQuestionTypeAndWaitForNewQuestionPanelReady(String questionType) {
        click(browser.driver.findElement(By.cssSelector("[data-questionType=" + questionType + "]")));

        waitForNewQuestionPanelReady();
    }
```

Maybe use this function itself and pass `TEMPLATE` as parameter. It should work.

I am against `scrollElementToCenterAndClick` cause it might not behave properly for `ChromeDriver`

Reply:
@tshradheya I'm curious what you meant by `scrollElementToCenterAndClick` not behaving properly in `ChromeDriver`? In fact, shouldn't it be the preferred way if we need to scroll to the element?

Not to mention using `click` (as opposed to `element.click()`) is a bad idea?

Reply:
> I'm curious what you meant by scrollElementToCenterAndClick not behaving properly in ChromeDriver

Oops. I think I misunderstood what was mentioned in the header comment of the function. 

@jacoblipech Maybe you can stick to using scrollElementToCenterAndClick() :p

Reply:
@tshradheya hmm do you have suggestions to improve the header comment?